airflow/cli/commands/version.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Version command"""
+from rich.console import Console
+
+import airflow
+from airflow.cli import airflow_cmd
+
+
+@airflow_cmd.command('version')
+def version():
+    """Displays Airflow version at the command line"""
+    console = Console()
```

Comment:
New Breeze already uses rich click. Maybe it should also use the same architecture, ie define console as singleton with configured theme/settings. See here https://github.com/apache/airflow/blob/main/dev/breeze/src/airflow_breeze/utils/console.py

Reply:
Yeah. I think it's a good idea to setup a consistent communication approach -> i.e. have custom "message type" tags rather than color tags - having that and documenting why and how to use them (like we did here) https://github.com/apache/airflow/blob/main/dev/breeze/doc/adr/0011-unified-communication-with-the-users.md - brings consistent user experience. And introducing themes  makes it easy to switch to a "colour-blind-friendly" mode from the beginning.

There are 8% males with red/green collour defficiency https://en.wikipedia.org/wiki/Color_blindness and males are (for better or worse) most of the airflow users, so it might make sense to implement a consistent "theme" approach and ability to switch, especially that it is rather easy and except the need of retrieving console dynamically, does not bring too much of an overhead in the long run.

Reply:
> Yeah. I think it's a good idea to setup a consistent communication approach -> i.e. have custom "message type" tags rather than color tags - having that and documenting why and how to use them (like we did here) https://github.com/apache/airflow/blob/main/dev/breeze/doc/adr/0011-unified-communication-with-the-users.md - brings consistent user experience. And introducing themes makes it easy to switch to a "colour-blind-friendly" mode from the beginning.
> 
> There are 8% males with red/green collour defficiency https://en.wikipedia.org/wiki/Color_blindness and males are (for better or worse) most of the airflow users, so it might make sense to implement a consistent "theme" approach and ability to switch, especially that it is rather easy and except the need of retrieving console dynamically, does not bring too much of an overhead in the long run.

Sure, but I would not do that in this PR, but  convert first, then add theme support as a follow up. a) Focus on one thing at a time, and b) that can be done in parallel to the migration work.

To make it easier we could have


```python
@cache
def get_console() -> Console:
    return Console()
```

in `airflow/cli/__init__.py`, just so that everything in the CLI uses that singleton already.